improver/calibration/ensemble_calibration.py

```diff
@@ -1366,6 +1367,61 @@ def process(
         return coefficients_cubelist
 
 
+class MetaEstimateCoefficientsForEnsembleCalibration(BasePlugin):
```

Comment:
This is a beast of a name 😋 
How about this:
```suggestion
class MetaEnsembleCoeffEstimator(BasePlugin):
```